python/ray/autoscaler/_private/util.py

```diff
@@ -575,11 +596,15 @@ def parse_usage(usage: Usage) -> List[str]:
 
         if resource in ["memory", "object_store_memory"]:
             to_GiB = 1 / 2**30
-            line = f"{(used * to_GiB):.2f}/" f"{(total * to_GiB):.3f} GiB {resource}"
+            formatted_used = format_memory(used)
+            formatted_total = format_memory(total)
+            line = f"{formatted_used}/{formatted_total} {resource}"
             if used_in_pg:
+                formatted_pg_used = format_memory(pg_used)
+                formatted_pg_total = format_memory(pg_total)
                 line = line + (
-                    f" ({(pg_used * to_GiB):.2f} used of "
-                    f"{(pg_total * to_GiB):.2f} GiB " + "reserved in placement groups)"
+                    f" ({formatted_pg_used} used of "
+                    f"{formatted_pg_total} GiB " + "reserved in placement groups)"
```

Comment:
This GiB will be extra now?
```suggestion
                    f"{formatted_pg_total} " + "reserved in placement groups)"
```